data/data_test.go

```diff
@@ -0,0 +1,132 @@
+package data
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+)
+
+type TestClient struct {
```

Comment:
Fixture types and functions should be lowercase. They won't be exported if they are uppercase but keeping them lowercase reduces confusion.

Reply:
Coupled with `package data` in this file, that was an unintended side-effect.